Index.py

```diff
@@ -65,4 +66,22 @@ def update(id: int):
         return make_response("the data updated", status.HTTP_200_OK)
 
 
+# *************************insert file Api***********************************#
+@app.route("/song_as_file", methods=["POST"])
+def insert_file():
+    my_contant = {}
+    my_contant["my_file"] = request.files.get("data_file")
+    my_contant["tag"] = request.form.get("tag")
+    my_contant["force_replace"] = request.form.get("force_replace")
+    if not contanet_validation.is_valid(my_contant):
+        return make_response("Bad request", status.HTTP_400_BAD_REQUEST)
+    else:
+        my_contant = contanet_validation.contatnt_make_valid(my_contant)
+        if not read_from_file.read_from_file_api(
+            my_contant["my_file"], my_contant["tag"], my_contant["force_replace"]
+        ):
+            return make_response("Bad Request", status.HTTP_400_BAD_REQUEST)
```

Comment:
Give more information in-place of the dots please..
```suggestion
            return make_response("......", status.HTTP_400_BAD_REQUEST)
```